spec/factories/rewards.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :reward do
+    sequence(:title) { |i| "Title#{i}" }
+    sequence(:description) { |i| "Description#{i}" }
+    sequence(:price) { |i| i }
+    title { 'MyString' }
+    description { 'MyText' }
+    price { '9.99' }
```

Comment:
I  think this is unnecessary - sequence helpers are covering title, description, and price

Reply:
leftover, forgot to delete

Reply:
Actually I have no clue how would the model created with that factory look like - I mean would it have attributes from sequence or regular assignment? Maybe it is worth checking and sharing with group as TIL?

Reply:
it was generated by using `rails g model reward title:string description:text price:decimal`, and after this we are getting model, migration, empty test spec, and factory that looks like these deleted lines :)

Reply:
I mean that you have all attributes defined twice - normal way (generator) and with sequence.

So when you create an object with that factory which attributes will it have?

Reply:
ohh thats because i was editing this factory by pushing down old definitions and creating sequences to make sure everything is correctly covered, and then i forgot to delete this unnecessary lines

Reply:
ok, let me rephrase the question

we have a (simplified) factory like this:

```
factory :reward do
    sequence(:title) { |i| "Foo_#{i}" }
    title { 'Bar' }
end
```

then if I create the reward object with that factory:

`let!(new_reward) {create(:reward)}`

what value will `new_reward.title` have?

As previously mentioned - I do not know, but maybe you want to investigate it, and share with the group :)